modules/income_and_assets/lib/income_and_assets/pdf_fill/forms/va21p0969.rb

```diff
@@ -559,6 +565,52 @@ class Va21p0969 < FormBase
             question_suffix: '(m)',
             question_text: 'DO YOU HAVE ANY ADDITIONAL AUTHORITY OR CONTROL OF THE TRUST?'
           }
+        },
+        # 10a
+        'unreportedAsset' => { key: 'F[0].#subform[8].DependentsHaveAssetsNotReported10a[0]' },
+        'unreportedAssets' => {
+          limit: 1,
+          first_key: 'assetOwnerRelationship',
```

Comment:
This key should be different: if you add a key for `otherRecipientRelationshipType` or something similar, then `first_key` should be set to that key